bridges/otellogr/logsink.go

```diff
@@ -139,6 +143,18 @@ func WithLoggerProvider(provider log.LoggerProvider) Option {
 	})
 }
 
+// WithContext returns an [Option] that configures the [context.Context] used by
+// a [LogSink].
+//
+// By default if this Option is not provided, the LogSink will use the
+// background context.
+func WithContext(ctx context.Context) Option {
```

Comment:
This is a different change. Please keep this PR constrained to fixing the nil panic, without the additional way to set the context.

Reply:
No problem~ I'll adjust it later.